pkg/estimator/client/service_test.go

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright 2022 The Karmada Authors.
```

Comment:
2024

Reply:
@XiShanYongYe-Chang fixed the required changes .